e2e-tests/playwright/tests/functional/system_console/sytem_users/column_sort.spec.ts

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {expect} from '@playwright/test';
+
+import {test} from '@e2e-support/test_fixture';
+import {createRandomUser} from '@e2e-support/server';
+
+const EMAIL_COLUMN = 'Email';
+const LAST_LOGIN_COLUMN = 'Last login';
```

Comment:
Since these are UI text and not from user input, I'd suggest to move this out of spec file into the "ui/component".